Packs/PassiveTotal/ReleaseNotes/2_0_5.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### PassiveTotal v2
+**Breaking Change** pt-get-pdns-details, pt-whois-search and pt-get-components commands changed to return multiple entries (entry per indicator) instead of a single entry.
```

Comment:
```suggestion
**Breaking Change:** ***pt-get-pdns-details***, ***pt-whois-search*** and ***pt-get-components*** commands were changed to return multiple entries (entry per indicator) instead of a single entry.
```